### PR TITLE
closes #46: Added jshint for .js & .ios.js

### LINKED ISSUE #46: Adding grunt.js for jshint
Working on getting jshint running on grunt. This will be linting only web files (server/client) and not (app), because app is written in react-native.

### PR DESCRIPTION
**Pull request by arianf**
- [x] Have `grunt test`, run jshunt for `.js` files
- [x] Have `grunt test`, also run jsxhint for `.ios.js` files

### CODE REVIEW

**Comment by rkho** [comment]
Looks great!

**Comment by koziscool** [comment]
lgtm

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
   "homepage": "https://github.com/arianf/crewapp",
   "dependencies": {
     "grunt": "^0.4.5",
-    "grunt-shell": "^1.1.2"
+    "grunt-jsxhint": "^0.6.0",
```
Comment (pravipati) [inline package.json:20]: Please add these to dev-dependencies.

Comment (arianf) [inline package.json:20]: We need these for production, so they aren't dev-dependencies.

**Pull request merged**